lxd/remote.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/lxc/lxd/shared"
+)
+
+func remoteGetFingerprint(d *Daemon, server string, alias string) (string, error) {
```

Comment:
Can we rename to remoteGetImageFingerprint or something like that? I first thought it was about getting the remote server cert fingerprint.